api/policies_test.go

```diff
@@ -0,0 +1,201 @@
+package api_test
```

Comment:
Consider adding a test case for when an error is returned from the permissions store.